### PR TITLE
removed template source code for generated template classes when debug is disabled

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

**Comment by mvrhov** [comment]
Thanks.

File: lib/Twig/Lexer.php [hunk 1]
```diff
@@ -136,7 +136,7 @@ public function tokenize($code, $filename = null)
             mb_internal_encoding($mbEncoding);
         }
 
-        return new Twig_TokenStream($this->tokens, $this->filename, $code);
+        return new Twig_TokenStream($this->tokens, $this->filename, $this->env->isDebug() ? $code : '');
```
Comment (stof) [inline lib/Twig/Lexer.php]: should it really be removed from the TokenStream itself ? IMO, this check belongs to the compilation step instead

Comment (LionsAd) [inline lib/Twig/Lexer.php]: I agree, it breaks my code that worked fine in 1.25.

**Comment by nicolas-grekas** [comment]
See #2169

**Pull request merged**